plugins/orchestrator/package.json

```diff
@@ -58,6 +58,7 @@
     "@rjsf/material-ui-v5": "npm:@rjsf/material-ui@5.7.3",
     "@rjsf/utils": "5.7.3",
     "@rjsf/validator-ajv8": "5.7.3",
+    "dataclass": "^2.1.1",
```

Comment:
I see that you are replacing the approach you've implemented yesterday with this new library.
Is it really necessary? What gains should we expect by using this library in favor of your previous implementation?
Just asking because I don't see much traction on their [repository](https://github.com/alexeyraspopov/dataclass).

Reply:
@jkilzi

Reply:
Wouldn't using [readonly](https://www.typescriptlang.org/docs/handbook/utility-types.html#readonlytype) to your type be enough?

Reply:
OK I'll add the readonly to the previous solution
We can close this PR